engine/core.py

```diff
@@ -134,6 +134,18 @@ def get_key_points(self, name: Optional[str] = None) -> Iterable[scripts.KeyPoin
 
         return self.world.get_key_points(name)
 
+    def get_sprites(self, name: Optional[str] = None) -> Iterable[arcade.Sprite]:
+        """Gets all sprites with the given name."""
+        if self.world is None:
+            raise GameNotInitializedError()
+
+        matching_sprites = []
+        for n in self.world.scripted_objects.items():
```

Comment:
It's not usually good practice to use single-variable names in Python. Since the key of the dictionary is the name, `obj_name` is better.